### ISSUE TITLE
Mushroom icon state doesn't correspond to "active_state:"

### ISSUE THREAD

**Issue by kasabov** [rianadon/timer-bar-card#94]
**Describe the bug**
The icon state in a mushroom row is not active while the timer itself is. Imho both states should be the same at all times.

I have a friend Jeardene who visits me occasionally. I track his visits in a calendar and render the remaining time to his next visit using the timer-bar-card. I also use:
* an input_datetime helper to track his last visit (via an automation using calendar event) and use it for the "start_time".
* a card-templater to display the remaining time in days in the "name_template". I couldn't figure out how to do this with the timer-bar-card alone.

**Your card configuration**
```yaml
type: custom:card-templater
card:
  mushroom:
    icon_color: accent
  sync_issues: ignore
  type: custom:timer-bar-card
  entity: calendar.jeardene
  active_state: 'off'
  start_time:
    entity: input_datetime.jeardene_last_visit
  invert: true
  bar_direction: rtl
  name_template: >-
    Jeardene visits again in {{ (as_timestamp(state_attr('calendar.jeardene',
    'start_time')) - as_timestamp(now()) ) | timestamp_custom("%j") |int }} days
  text_width: 6em
  icon: mdi:broom
  debug: true
...
```

**Debug information**

<img width="492" alt="Screenshot 2023-07-04 at 23 27 03" src="https://github.com/rianadon/timer-bar-card/assets/10500163/5405e122-e14b-43e0-910f-cd3bb1c4a71e">

I'd like the broom icon to be in the same yellow (accent) colour as the timer progress bar is.

**Comment by rianadon** [comment]
Good point. And that's a great use of input_datetime and the calendar! I was using the mushroom card code to determine when the icon is active, but I should be really using what the timer bar card says.

The template looks great! You're right that it's not possible to use the card to show the time remaining in days, although that could be a good feature to add.

**Comment by kasabov** [comment]
> using the mushroom card code to determine when the icon is active

Right. From what I saw in the code, the icon uses the calendar entity for its active/disabled state, whereas it should rely on the "active_state" of the timer bar card. 

I also found it unintuitive that "mushroom.icon_color" sets the color of the progress bar too, instead of just the icon. Rename it altogether to "mushroom.color" perhaps.

> it's not possible to use the card to show the time remaining in days, although that could be a good feature to add

I understand. I don't mind not having that feature in order to keep the component simpler. You've documented how to combine it with the "template card" so that I can render whatever template I want and that worked well for me. The UI looks very cool indeed; good job.
